src/fabric-video/web/index.js

```diff
@@ -23,10 +27,33 @@ getIframeId()
     // We'll start the video when the slot is in view, but we want this
     // process to happen only once. When the video ends, we let everyone
     // know about it.
-    let video = document.getElementsByTagName('video')[0];
-    let played = false;
+    let video = document.getElementsByTagName('video');
+
+
+    if ( (VIDEO_MARCOS['videoUrl'] && VIDEO_MARCOS['mobileVideoUrl']) == '' ) return;
```

Comment:
Yes, my intention was only carrying on if we have both the desktop and mobile video urls.

Although this may be redundant as we can just make the fields within GAM mandatory.

Reply:
It might be a bit easier to parse with the following syntax:

`if( !VIDEO_MARCOS['videoUrl'] || !VIDEO_MARCOS['mobileVideoUrl']) return;`

This will stop execution if either the videoUrl are missing (`undefined`) or an empty string (`""`).